### ISSUE TITLE
A question about dataset

### ISSUE THREAD

**Issue by WhiteNan** [Encryptional/SOFB#1]
Thank you for developing this model.
I found that in the dataset you are using, there are two numerical sequences under a protein sequence in the training set, while there is only one sequence in the testing set. I want to know what this number represents and why the training and testing sets are different in this regard.

**Comment by Encryptional** [comment]
Thank you so much for studying the article!

The two number sequences under the protein sequences in the training set are residue labels. The first row of labels represents the original labels, while the second row of labels represents the labels obtained by amplification annotation from similar protein sequences. This method relieves the problem of positive and negative sample imbalance. In the test set, the protein data is only used to validate the performance of the model. To ensure fairness in validation, the test set does not need to be amplified and can be tested using only the original tags.

Hope my paper can help you with your research! If you have any other questions, please feel free to contact me.


> 2024年11月9日 12:53，WhiteNan ***@***.***> 写道：
> 
> two numerical sequences under a protein sequence in the training se

**Comment by WhiteNan** [comment]
Thanks for your reply!
I want to know how it is done when dividing the training set. Is it to combine a protein sequence with the following two sequences separately? Will the following numerical sequence be input into the model for prediction?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Encryptional/SOFB"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年11月9日(星期六) 中午1:23
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Encryptional/SOFB] A question about dataset (Issue #1)





 Thank you so much for studying the article! <br&gt; <br&gt; The two number sequences under the protein sequences in the training set are residue labels. The first row of labels represents the original labels, while the second row of labels represents the labels obtained by amplification annotation from similar protein sequences. This method relieves the problem of positive and negative sample imbalance. In the test set, the protein data is only used to validate the performance of the model. To ensure fairness in validation, the test set does not need to be amplified and can be tested using only the original tags. <br&gt; <br&gt; Hope my paper can help you with your research! If you have any other questions, please feel free to contact me. <br&gt; <br&gt; <br&gt; &amp;gt; 2024年11月9日 12:53，WhiteNan ***@***.***&amp;gt; 写道： <br&gt; &amp;gt;  <br&gt; &amp;gt; two numerical sequences under a protein sequence in the training se <br&gt; <br&gt;

**Comment by Encryptional** [comment]
The training process uses the augmented labels and will not use the original labels of the training set. The details of the processing of the training set can be learned from the code.
